### ISSUE TITLE
A11y_SQL Azure Data Studio_New Terminal_Keyboard:Tab key or shift+tab does not move to link "https://aka.ms/pscore6" is not working.

### ISSUE THREAD

**Issue by Roopesh-T** [microsoft/vscode#182849]
“[Check out Accessibility Insights](https://accessibilityinsights.io/)! – Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev1;#BM-VisualStudioCode-Win32-Nov2021#Visual Studio Code Client;#Win11;#FTP;#DesktopApp;#WCAG2.1.1;#Benchmark;#Keyboard;#AINotInScope;#AILimited;

### Environment Details:
Application Name: SQL Azure Data Studio
Version: 1.45.0-insider (user setup)
Windows Version: Windows11

### Repro Steps:
1. Launch Azure Data Studio Insiders application.
2. Tab to server drop down and tab again to button new connection press enter key and add the server details.
3. Tab to view menu item in the menu bar and use down key arrow to select item "terminal"
4. Use shift+tab to navigate to https://aka.ms/pscore6

### Actual:
On navigating using shift+tab or tab to link https://aka.ms/pscore6 keyboard key tab cannot access it.

### Expected:
Tab and shift+tab from the first line of command should move to link https://aka.ms/pscore6

### User Impact:
Keyboard users will not be able to access the link.

### Attachment:
[22333_A11y_SQL.Azure.Data.Studio_New.Terminal_Keyboard.zip](https://github.com/microsoft/azuredatastudio/files/10979474/22333_A11y_SQL.Azure.Data.Studio_New.Terminal_Keyboard.zip)

**Comment by Roopesh-T** [comment]
primary bug:https://github.com/microsoft/azuredatastudio/issues/22333

**Comment by alanrenmsft** [comment]
OS: Windows
Repro steps: 
1. open terminal tab
2. add a powershell terminal
3. run the command `powershell`

Expected: the link should be accessible using keyboard

![terminal](https://github.com/microsoft/azuredatastudio/assets/13777222/a6f09e37-5c67-4d98-a732-e62755df237f)

**Comment by meganrogge** [comment]
I don't think `Shift+tab` has ever been the way we access links in the terminal, but @Tyriar pls correct me if I'm wrong. 

We have a command that makes opening links keyboard accessible: `Terminal: Open Detected Link`
![Screenshot 2023-05-18 at 12 43 41 PM](https://github.com/microsoft/vscode/assets/29464607/cb8d5db9-24e1-4fa5-b746-10cf8c7e8f40)

**Comment by Tyriar** [comment]
Yep, here's the docs that call that out https://code.visualstudio.com/docs/terminal/basics#_links

**Comment by alanrenmsft** [comment]
@Tyriar @meganrogge thanks!

**Comment by meganrogge** [comment]
thanks for the gif 👍🏼

**Comment by Roopesh-T** [comment]
Closing the bugs as we are able to access the links in the terminal using command

We have a command that makes opening links keyboard accessible: Terminal: Open Detected Link
![22333_Closed](https://github.com/microsoft/azuredatastudio/assets/86601127/bd120dc1-4e6d-4b3f-b151-99e447e1fee8)
GitHub Tags : #Closed;#regressed:-05-22-23
